public/recentMastery.php

```diff
@@ -1,67 +1,66 @@
 <?php
 
+use App\Community\Enums\AwardType;
+use App\Platform\Enums\UnlockMode;
+use Illuminate\Support\Facades\Blade;
+
 authenticateFromCookie($user, $permissions, $userDetails);
 
 $maxCount = 25;
 // First day with Game Awards
-$minDate = '2014-09-29';
+$minDate = '2014-08-22';
```

Comment:
I changed this based on what I see in the DB. But it may be worth just changing this to 2013-03-02.